libraries/AP_RangeFinder/RangeFinder.h

```diff
@@ -182,6 +183,14 @@ class RangeFinder
      */
     bool pre_arm_check() const;
 
+    // return a 3D vector defining the position offset of the sensor in metres relative to the body frame origin
+    const Vector3f get_pos_offset(uint8_t instance) const {
```

Comment:
The const says that the function doesn't change the object. *instance* could also be const since we don't change it, but there's no need for that.